edgelet/iotedge/src/system.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use std::ffi::OsStr;
+
+use lazy_static::lazy_static;
+
+use aziotctl_common::{
+    get_status, get_system_logs as logs, restart, set_log_level as log_level, ServiceDefinition,
+    SERVICE_DEFINITIONS as IS_SERVICES,
+};
+
+use crate::error::{Error, ErrorKind};
+
+lazy_static! {
+    static ref IOTEDGED: ServiceDefinition = {
+        let sockets: &'static [&'static str] = option_env!("IOTEDGE_HOST").map_or(
+            &["aziot-edged.mgmt.socket", "aziot-edged.workload.socket"],
+            |_host| &[],
+        );
+
+        ServiceDefinition {
+            service: "aziot-iotedgd.service",
```

Comment:
`aziot-edged.service`

(You didn't test this, did you? :) )

Reply:
I tested for the aziot commands, but I didn't have a 1.2 edge install handy 😅

Reply:
ok, I installed a current version of iotedge 1.2 and tested the commands. It all works, but it brought an interesting question: 

If iotedge fails to start, it will never start IS. This means if both edged and identityd are unable to start, status wouldn't start reporting the identityd failure until edged was fixed (b/c identityd is never started).

I think this is fine, but its something to note maybe

Reply:
If edged doesn't start specifically because it crashes before trying to start IS, then that's likely an edged-specific problem anyway. And if edged doesn't start because it couldn't get IS to start, then both would be `failed`. So it should be fine.